Source: git show 22ba47f5
Upstream: in master
Reason: fix git-svn tests with subversion 1.5.5

commit 22ba47f544a233958c73a9260b18c57b431f44b3
Author: Marcel Koeppen <git-dev@marzelpan.de>
Date:   Mon Jan 19 03:02:01 2009 +0100

    git-svn: Show UUID in svn info for added directories with svn 1.5.5
    
    In svn 1.5.5 the output of "svn info" for added directories was changed
    and now shows the repository UUID. This patch implements the same
    behavior for "git svn info" and makes t9119-git-svn-info.17 pass if
    svn 1.5.5 is used.
    
    Signed-off-by: Marcel Koeppen <git-dev@marzelpan.de>
    Acked-by: Eric Wong <normalperson@yhbt.net>

diff --git a/git-svn.perl b/git-svn.perl
index 55c4dfb..d4cb538 100755
--- a/git-svn.perl
+++ b/git-svn.perl
@@ -912,7 +912,8 @@ sub cmd_info {
 	if ($@) {
 		$result .= "Repository Root: (offline)\n";
 	}
-	$result .= "Repository UUID: $uuid\n" unless $diff_status eq "A";
+	$result .= "Repository UUID: $uuid\n" unless $diff_status eq "A" &&
+		($SVN::Core::VERSION le '1.5.4' || $file_type ne "dir");
 	$result .= "Revision: " . ($diff_status eq "A" ? 0 : $rev) . "\n";
 
 	$result .= "Node Kind: " .
